src/sync.cpp

```diff
@@ -680,23 +680,22 @@ Sync::Sync(UnifiedSync& us, const char* cdebris,
     scanseqno = 0;
 
     mLocalPath = mUnifiedSync.mConfig.getLocalPath();
-    LocalPath crootpath = LocalPath::fromPath(mLocalPath, *client->fsaccess);
```

Comment:
needs adjustment for APPLE (code below)

Reply:
Hmm, I don't see why that would make a difference for APPLE, but I don't see the change causing any problems - I'll apply those, thanks

Reply:
It was the code inside the `#ifdef __APPLE__`, that needs refactoring of the variable. I'll fix it ;)

Reply:
Fixed in e6f6647